lib/sql-string.js

```diff
@@ -193,14 +193,16 @@ SqlString.bufferToString = function(buffer, dialect) {
 };
 
 SqlString.objectToValues = function(object, timeZone) {
-  var values = [];
-  for (var key in object) {
```

Comment:
There's a bunch of these changes, can you provide reasoning?
I'm aware that `for in` will break on changed `Object` prototypes, but the change to a for loop really lowers readability.